pr/README.md

```diff
@@ -7,6 +7,149 @@ file through a Pull Request.
 
 ## Usage
 
+```yml
+name: Tooling
+on:
+  workflow_dispatch: ~
+
+permissions: read-all
+
+jobs:
+  tooling:
+    name: Update tooling
+    uses: ericcornelissen/tool-versions-update-action/.github/workflows/pr.yml@v2
```

Comment:
The permissions for the job are missing, unlike [the base action](https://github.com/ericcornelissen/tool-versions-update-action/blob/dd7fb43852e8ab8bba8f07a832941787a21f4925/README.md#permissions), this action requires [write access to `contents` and `pull-requests`](https://github.com/ericcornelissen/tool-versions-update-action/blob/dd7fb43852e8ab8bba8f07a832941787a21f4925/pr/README.md#permissions).

```suggestion
permissions: read-all

jobs:
  tooling:
    name: Update tooling
    uses: ericcornelissen/tool-versions-update-action/.github/workflows/pr.yml@v2
    permissions:
      contents: write # To push a commit
      pull-requests: write # To open a Pull Request
```